### ISSUE TITLE
Accessibility: Filters missing status message

### ISSUE THREAD

**Issue by vikrampm1** [woothemes/woocommerce#52090]
**Prerequisites (mark completed items with an [x]):**
- [x] I have checked that my issue type is not listed here https://github.com/woocommerce/woocommerce/issues/new/choose
- [x] My issue is not a security issue, support request, bug report, enhancement or feature request (Please use the link above if it is).

### Issue Description:

When filters are submitted or removed or all filters are cleared, there is not auditory success message or any way for blind users to know that the product list has changed without exploring the page.

There is a visual message that shows the number of results found but it is not programatically defined as a status message so it's not announced. Also, this Product Results Count block can be removed by content creators.

### Component or Page: 

Components: Filter by Attribute block, Filter by price block, Active Filters Block, Rating Filters block

### Name affected element:

All filters and reset, apply, and clear all buttons

### Relevant code snippets:

n/a

### Recommended fix:

Add a section to the search results that tells how many results were found. This can be visible for all users using the Results Count block, but only if you force it always exist on the page and block content creators from removing it. If that's not possible, add a hidden alert container for screen reader users only. As searches are submitted or filters cleared, change the content in this container to reflect the number of results for each search. Add role=""alert"" and aria-live=""polite"" to ensure that the results count is announced to screen reader users.
 
 Learn more about the alert role: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/alert_role 

Sometimes the alert role is difficult to always get read out when there is a page refresh. An alternate way to convey similar information about results to to add them into the <title> tag for the page. I.e., <title> Shop (1-10 of 200 products) | Site Name</title> Then change the result count in the title tag as the results change. Then, when the page reloads and reads out the title, the count will be conveyed.

**Comment by Manussakis** [comment]
Hey @gigitux 👋

Please confirm if this issue can be closed since it's related to the old filter blocks that you are no longer investing in.
https://github.com/woocommerce/woocommerce/pull/53681#issuecomment-2598743908

Thanks!

cc: @vikrampm1

**Comment by gigitux** [comment]
Thanks for the ping! Yes, we can close this issue!

**Comment by amberhinds** [comment]
@gigitux for the VPAT, what are alternative blocks that users can switch to that that resolves this issue?

**Comment by gigitux** [comment]
`Product Filters` block that will be available in WooCommerce 9.9 (next release).

**Comment by Manussakis** [comment]
Tagging @vikrampm1 for visibility of the comment above.
